haproxy/variables.tf

```diff
@@ -50,3 +50,11 @@ variable "backends_weights" {
     },
   ]
 }
+
+variable "cidr_block" {
+  default = "100.127.0.0/16"
+}
+
+variable "dns_ip" {
+  default = "100.127.0.2"
```

Comment:
You can also use this https://www.terraform.io/docs/configuration/interpolation.html#cidrhost-iprange-hostnum-

Reply:
Bleh. `Error: variable "dns_ip": default may not contain interpolations`

Reply:
Yea you can't do that but you can drop the var and feed it into the template directly ;)